src/components/missionControl/EditingWorkspace.tsx

```diff
@@ -151,7 +162,7 @@ class AssessmentWorkspace extends React.Component<AssessmentWorkspaceProps, ISta
     const questionId = this.formatedQuestionId();
     const question: IQuestion = this.state.assessment.questions[questionId];
     const workspaceProps: WorkspaceProps = {
-      controlBarProps: this.controlBarProps(this.props, questionId),
+      controlBarProps: this.controlBarProps(this.props, this.state, questionId),
```

Comment:
Why is the state being passed into the class function? Just reference it from within the class

Reply:
There was actually a quirk with this function in which it took props as an argument, but the body referenced `this.props` instead of the parameter `props`

Not really sure what was originally intended but I fixed it to use the props that were passed into it, and then I decided that the state could be handled this way as well for consistency

Would it be a good idea to just not pass either props or state? Or is there a reason to pass only props

Reply:
I think the props shouldn't be passed to. The class function can reference the props and state using `this`

Reply:
Ok, this can be done. I suppose it can be changed to be this way for `AssessmentWorkspace`, `GradingWorkspace` as well?